docs/flashbots-auction/searchers/advanced/private-transaction.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: private transactions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## How to send a single transaction to Flashbots
+
+If you want to send a single transaction to flashbots, without sending it as a bundle, you can use the `sendPrivateTransaction` method.
+
+This method has a few key advantages:
+
+1. This method attempts to send your transaction to miners on every block for a maximum of 25 blocks. No need to listen for the next block and re-send yourself. Just call `result.wait()` and check the result.
```

Comment:
"Just call `result.wait()` and check the result." Not everyone will use js! Suggest we remove that. Rest looks good.

Reply:
ah right, good point. removed.